src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -238,7 +238,7 @@ const SignOrExecuteForm = ({
 
         <TxSimulation
           gasLimit={advancedParams.gasLimit?.toNumber()}
-          transactions={safeTx}
+          transactions={tx}
```

Comment:
Oh, damn, I can see why this was easy to miss. Maybe we should rename `safeTx` to `_originalTx` or something?